lib/ipmi_tsol.c

```diff
@@ -534,7 +534,7 @@ ipmi_tsol_main(struct ipmi_intf *intf, int argc, char **argv)
 
 		if ((fds[0].revents & POLLIN) && (sizeof(out_buff) > out_buff_fill)) {
 			socklen_t sin_len = sizeof(sin);
-			int buff_size = sizeof(buff);
+			unsigned short int buff_size = sizeof(buff);
```

Comment:
The type of `sizeof()` is `size_t`, please use it. Please remember that there are different architectures too.

Reply:
> The type of `sizeof()` is `size_t`, please use it. Please remember that there are different architectures too.

Thanks, done.